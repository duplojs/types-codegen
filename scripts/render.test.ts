import { render } from "./render";

it("render", () => {
	expect(
		render("test"),
	).toMatchSnapshot();
});
