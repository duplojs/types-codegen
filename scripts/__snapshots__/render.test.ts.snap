// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`render 1`] = `
"// Generated by @duplojs/types-codegen
/* eslint-disable */
/* prettier-ignore */
/* istanbul ignore file */
/* v8 ignore start */
// noinspection JSUnusedGlobalSymbols
// @ts-nocheck
test
/* v8 ignore stop */
"
`;
